packages/api-plugin-fulfillment-method-pickup-store/src/util/validateOrderMethodsstore.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @summary Sample dummy function validating the data requirements for this metod
+ * @param {Object} context - Context object
+ * @param {Object} commonOrder - Current order which provide available details to perform validation
+ * @param {Object[]} validationResults - Validation results collected till now
+ * @returns {Object[]} validationResults - with the validation details populated
+ */
+export default function validateOrderMethodsstore(context, commonOrder, validationResults = []) {
```

Comment:
function violates naming convention, must be camel-case

Reply:
Fixed by converting the ff-type to Title case before concatenating